core/common/src/main/java/alluxio/resource/ResourcePool.java

```diff
@@ -153,6 +160,14 @@ public void release(T resource) {
   }
 
   /**
+   * @return the current pool size
```

Comment:
NIT: for overriden methods, you don't need to define its javadoc anymore, unless there is something more specific than the javadoc in its super class or interface.